### ISSUE TITLE
Small bug in get_load() of ODBC returner

### ISSUE THREAD

**Issue by djbeadle** [saltstack/salt#49810]
**Issue:**
In the *get_load* function of the ODBC returner, the following return fails producing a TypeError with the message *"expected string or buffer"*
```python
if data:
    return salt.utils.json.loads(data)
```

**Full source of function:**
```python
def get_load(jid):
    '''
    Return the load data that marks a specified jid
    '''
    conn = _get_conn(ret=None)
    cur = conn.cursor()
    sql = '''SELECT load FROM jids WHERE jid = ?;'''

    cur.execute(sql, (jid,))
    data = cur.fetchone()
    if data:
        return salt.utils.json.loads(data)
    _close_conn(conn)
    return {}
```

Link to function: https://github.com/saltstack/salt/blob/b44f0f1d0fd564690c6e45c6dd061c0e84d76def/salt/returners/odbc.py#L233

**Proposed Solution:**
Change:
```python
if data:
    return salt.utils.json.loads(data)
```
to
```python
if data:
    return salt.utils.json.loads(str(data))
```

**Comment by garethgreenaway** [comment]
@djbeadle thanks for the report.  can you test the above changes with `salt.utils.stringutils.to_str`?

**Comment by djbeadle** [comment]
@garethgreenaway I revisited this with fresh eyes after the weekend. 

I now believe the proper solution is to simply change: https://github.com/saltstack/salt/blob/b44f0f1d0fd564690c6e45c6dd061c0e84d76def/salt/returners/odbc.py#L244 

to:
```python
 return salt.utils.json.loads(data[0]) 
```

The issue is that fetchone() returns an array with a single item in it. I failed to notice the above change that my coworker had made-- the string coercion was simply a red herring.
